src/lib/components/ui/Button/Button.jsx

```diff
@@ -48,8 +49,15 @@ export const Button = (props) => {
     labelVisibilityClass = styles.withLabelHidden;
   }
 
+  const propsToTransfer = transferProps(
+    props,
+    ['afterLabel', 'beforeLabel', 'block', 'clickHandler', 'disabled', 'endCorner', 'grouped', 'id',
```

Comment:
Cannot the second argument simply be `Object.keys(propTypes)`? If I understand it correctly, we always want to transfer all props except `class` and `forwardedRef`, maybe `children`. I'm just pointing this out because it seems to me we should be able to get to the same outcome in a more DRY way.

Reply:
We have discused it over mobile. We will leave it now as it is and rework it earlier. We will need to found out way how to pass props into it